### ISSUE TITLE
When Support for Angular Material 17?

### ISSUE THREAD

**Issue by Skycader** [arciisine/MaterialThemeGenerator#70]
Your tools seems extremely well done, however it's been 2 years since last commit. why so?

**Comment by ben-wall** [comment]
Yeah it's the same isn't it I really liked this tool but it did stop working at some point.

**Comment by redevill** [comment]
The sass structure of Angular Material has changed quite a bit - and this nice tool no longer produces compatible output for Version 17

**Comment by arciisine** [comment]
Yeah, that is actually the core issue here.  For Angular 17, the Material library is essentially new.  This project would need to be rewritten to produce the new structure needed, but afaict not all the existing functionality would be supported either.

There is definitely a thought for upgrading to 17, but the level of effort here is substantially higher than the migration from all previous versions.
